docs/application/native/guides/text-input/autofill.md

```diff
@@ -0,0 +1,365 @@
+    char *app_id = NULL;
+
+    app_get_id(&app_id);
+
+    /* create autofill item for entering an user ID */
+    autofill_item_create(&ai_id);
+    autofill_item_set_autofill_hint(ai_id, AUTOFILL_HINT_ID);
+    autofill_item_set_id(ai_id, "id");
+    autofill_item_set_label(ai_id, "Enter your ID");
+    autofill_item_set_sensitive_data(ai_id, false);
+
+    /* create autofill item for entering an user password */
+    autofill_item_create(&ai_password);
+    autofill_item_set_autofill_hint(ai_password, AUTOFILL_HINT_PASSWORD);
+    autofill_item_set_id(ai_password, "password");
+    autofill_item_set_label(ai_password, "Enter your password");
+    autofill_item_set_sensitive_data(ai_password, true);
+
+    /* create autofill view info */
+    autofill_view_info_create(&vi_h);
+    autofill_view_info_set_app_id(vi_h, app_id);
+    autofill_view_info_set_view_id(vi_h, "login");
+
+    autofill_view_info_add_item(vi_h, ai_id);
+    autofill_view_info_add_item(vi_h, ai_password);
+    ```
+
+3. Request to get the data previously entered by the user and stored in an autofill service:
+
+    ```c
+    /* Send request to get autofill fill response */
+    int ret = autofill_fill_request(ah, vi_h);
+    if (ret != AUTOFILL_ERROR_NONE)
+        /* Error handling */
+    ```
+
+4. When you no longer need the autofill view handle, call `autofill_view_info_destroy()` to release autofill view handle:
+
+    ```c
+    autofill_view_info_destroy(vi_h);
+    ```
+
+## Requesting to store the data entered by the user in autofill service
+
+1. Create autofill data for each input field for saving:
+
+    ```c
+    autofill_save_item_h si_h;
+
+    /* create autofill item for saving */
+    autofill_save_item_create(&si_h);
+    autofill_save_item_set_autofill_hint(si_h, AUTOFILL_HINT_ID);
+    autofill_save_item_set_id(si_h, "id");
+    autofill_save_item_set_label(si_h, "Input your ID");
+    autofill_save_item_set_sensitive_data(si_h, false);
+    autofill_save_item_set_value(si_h, "myID");
+    ```
+
+2. Create autofill view data to include multiple autofill items:
+
+    ```c
+    char *app_id;
+
+    /* create autofill save view info */
+    app_get_id(&app_id);
+
+    autofill_save_view_info_create(&svi_h);
+    autofill_save_view_info_set_app_id(svi_h, app_id);
+    autofill_save_view_info_set_view_id(svi_h, "login");
+
+    /* append autofill save item in autofill save view */
+    autofill_save_view_info_add_item(svi_h, si_h);
+
+    if (app_id)
+        free(app_id);
+    ```
+
+3. Request to store the data in an autofill service using `autofill_commit()`:
+
+    ```c
+    int ret;
+
+    /* Send request to save autofill data */
+    ret = autofill_commit(ah, svi_h);
+    if (ret != AUTOFILL_ERROR_NONE)
+        /* Error handling */
+
+    autofill_save_view_info_destroy(svi_h);
+    ```
+
+4. When you no longer need the autofill save view handle, call `autofill_view_info_destroy()` to release autofill save view handle:
```

Comment:
@mijin85cho, Thank you pointing out.
`autofill_save_view_info_destroy()` is correct. Since it is part of the code.
@imfine98, please can you change the function name.

Reply:
Thanks for reporting this issue.
I've fixed.